tests/components/dwd_weather_warnings/test_config_flow.py

```diff
@@ -121,26 +117,20 @@ async def test_create_entry_gps(
         {ATTR_LATITUDE: "50.180454", ATTR_LONGITUDE: "7.610263"},
     )
 
-    with patch(
-        "homeassistant.components.dwd_weather_warnings.config_flow.DwdWeatherWarningsAPI",
-        return_value=False,
-    ):
-        result = await hass.config_entries.flow.async_configure(
-            result["flow_id"], user_input=DEMO_CONFIG_ENTRY_GPS
-        )
+    mock_dwdwfsapi.__bool__.return_value = False
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input=DEMO_CONFIG_ENTRY_GPS
+    )
 
     await hass.async_block_till_done()
     assert result["type"] == FlowResultType.FORM
     assert result["errors"] == {"base": "invalid_identifier"}
 
     # Test for successfully created entry.
-    with patch(
-        "homeassistant.components.dwd_weather_warnings.config_flow.DwdWeatherWarningsAPI",
-        return_value=True,
-    ):
-        result = await hass.config_entries.flow.async_configure(
-            result["flow_id"], user_input=DEMO_CONFIG_ENTRY_GPS
-        )
+    mock_dwdwfsapi.__bool__.return_value = True
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input=DEMO_CONFIG_ENTRY_GPS
+    )
 
     await hass.async_block_till_done()
     assert result["type"] == FlowResultType.CREATE_ENTRY
```

Comment:
```suggestion
    assert result["type"] is FlowResultType.CREATE_ENTRY
```

Reply:
Updated in f3c37a8f1a1a28d95dbb089be707fe055589725b